.nvmrc

```diff
@@ -1 +0,0 @@
-10.15.3
```

Comment:
Does it hurt to keep `.nvmrc` around?  We still have many node projects that only include this file, and `asdf` is compatible with it.